src/components/Button.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = (props) => {
+  const { name } = props;
+  return (
+    <div>
+      { name }
+    </div>
+  );
+};
+
+Button.defaultProps = {
+  name: '0',
+};
```

Comment:
we shouldn't have a button without a name so no need for defaultProps. instead lets add `required` to the propType so it cant be blank